From e6ea13fe4e59db906fa8f9f63f3ce6a951a1fea6 Mon Sep 17 00:00:00 2001
From: ewt45 <79033456+ewt45@users.noreply.github.com>
Date: Sat, 27 Sep 2025 13:48:34 +0800
Subject: [PATCH] 1

---
 build.gradle | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index e873241..6695150 100644
--- a/build.gradle
+++ b/build.gradle
@@ -13,7 +13,6 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:4.1.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:1.4.1'
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
-- 
2.43.0

